gs-web-elasticsearch/doc/index.rst

```diff
@@ -296,60 +296,170 @@ Grid Strategy
 ^^^^^^^^^^^^^
 The ``gridStrategy`` parameter identifies the ``mil.nga.giat.process.elasticsearch.GeoHashGrid`` implemenation that will be used to convert each geohashgrid bucket into a raster value (number).
 
-* ``basic``: returns top level ``doc_count``.
+Basic
+~~~~~
+Returns top level ``doc_count``.
 
-    Example Aggregation::
+Example Aggregation::
+
+  {
+    "agg": {
+      "geohash_grid": {
+        "field": "geo"
+      }
+    }
+  }
     
-      {
-        "agg": {
-          "geohash_grid": {
-            "field": "geo"
+Example bucket::
+
+ {
+   "key" : "xv",
+   "doc_count" : 1
+ }
+
+Extracted raster value: ``1``
+
+Metric
+~~~~~~
+Returns metric value.
+
+.. list-table::
+   :widths: 20 20 60
+
+   * - Parameter
+     - Default Value
+     - Description
+   * - Metric key
+     - ``metric``
+     - Key used to pluck metric object from top level bucket. Empty string results in plucking doc_count.
+   * - Value key
+     - ``value``
+     - Key used to pluck the value from the metric object.
+

```

Comment:
Since these aren't named parameters, what about changing "Parameter" to "Argument Index" and then use "0" and "1" instead of "Metric key" and "Value key"? Or else something else to make clear these parameters are for the `gridStrategyArgs` list?